README.md

```diff
@@ -95,3 +98,25 @@ When you're finished, remember to run:
 ```bash
 npm run infra:dev:stop
 ```
+
+## Contributors ✨
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/w3dani"><img src="https://avatars.githubusercontent.com/u/48931264?v=4?s=100" width="100px;" alt="w3dani"/><br /><sub><b>w3dani</b></sub></a><br /><a href="#infra-w3dani" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="https://github.com/guidesmiths/bamboohrbot/pulls?q=is%3Apr+reviewed-by%3Aw3dani" title="Reviewed Pull Requests">👀</a></td>
```

Comment:
<img src="https://media3.giphy.com/media/xT5LMFZDsj0AKUDYTS/giphy.gif"/>

Reply:
I'll reopen it, @w3dani . Don't worry! :)